input/examples/procedure-dialysis.xml

```diff
@@ -18,7 +15,7 @@
     <text value="Hemodialysis"/>
```

Comment:
Although SNOMED FSN is 'Hemodialysis (procedure), consider the use of AU preferred spelling/SNOMED preferred term for text value 'Haemodialysis'.

Reply:
Done

Reply:
No change in PR.

Reply:
Please check again